Flaky test everything integration test

```

....................s..F.............................s...s................sssss...sss..ss.........................................s.s..................................................................s..s..s..........................................s..s.s.....s.. [ 93%]
................s.                                                                                                                                                                                                                                                     [100%]
================================================================================================================================== FAILURES ==================================================================================================================================
______________________________________________________________________________________________________________________ test_everything_integration_test ______________________________________________________________________________________________________________________
[gw6] darwin -- Python 3.13.0 /opt/miniconda3/envs/swea13/bin/python

    @pytest.mark.slow
    async def test_everything_integration_test():
        app = AgentApp(
            model=DeterministicModel(
                outputs=[
                    "/sleep 0.5",
                    "THOUGHTT 1\n ```bash\necho '1'\n```",  # step 2
                    "THOUGHTT 2\n ```bash\necho '2'\n```",  # step 3
                    "THOUGHTT 3\n ```bash\necho '3'\n```",  # step 4
                    "THOUGHTT 4\n ```bash\necho '4'\n```",  # step 5
                    "THOUGHTT 5\n ```bash\necho '5'\n```",  # step 6
                    "THOUGHTT 6\n ```bash\necho '6'\n```",  # step 7
                    "FINISHING\n ```bash\necho 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'\n```",
                    "FINISHING2\n ```bash\necho 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'\n```",
                ],
            ),
            env=LocalEnvironment(),
            task="What's up?",
            mode="confirm",
            cost_limit=10.0,
        )
        assert app.agent.config.confirm_exit
        async with app.run_test() as pilot:
            assert app.agent_state == "RUNNING"
>           assert "You are a helpful assistant that can do anything." in get_screen_text(app)
E           AssertionError: assert 'You are a helpful assistant that can do anything.' in 'AgentApp\nWaiting for agent to start...'
E            +  where 'AgentApp\nWaiting for agent to start...' = get_screen_text(AgentApp(title='AgentApp', classes={'-dark-mode'}, pseudo_classes={'focus', 'dark'}))

tests/agents/test_interactive_textual.py:76: AssertionError
============================================================================================================================== warnings summary ==============================================================================================================================
tests/agents/test_interactive_textual.py::test_everything_integration_test
```

--- a/tests/agents/test_interactive_textual.py
+++ b/tests/agents/test_interactive_textual.py
@@ -72,6 +72,7 @@ async def test_everything_integration_test():
     )
     assert app.agent.config.confirm_exit
     async with app.run_test() as pilot:
+        await pilot.pause(0.2)
         assert app.agent_state == "RUNNING"
         assert "You are a helpful assistant that can do anything." in get_screen_text(app)
         assert "press enter" not in get_screen_text(app).lower()

